src/passes/Print.cpp

```diff
@@ -21,12 +21,22 @@
 #include <ir/module-utils.h>
 #include <pass.h>
 #include <pretty_printing.h>
-#include <wasm-printing.h>
 #include <wasm-stack.h>
 #include <wasm.h>
 
 namespace wasm {
 
+static std::ostream& printExpression(Expression* expression,
+                                     std::ostream& o,
+                                     bool minify = false,
+                                     bool full = false);
+
+static std::ostream&
+printStackInst(StackInst* inst, std::ostream& o, Function* func = nullptr);
+
+static std::ostream&
+printStackIR(StackIR* ir, std::ostream& o, Function* func = nullptr);
```

Comment:
Would it work to move these into the anonymous namespace below (and do the same for their definitions) rather than mark them `static`?

Reply:
Not easily... the anonymous namespace starts and ends pretty early in the file, but these impls need stuff all the way at the end. So it might take a lot of code motion to use an anonymous namespace for this.

Another option might be to use a non-anonymous namespace for them, but that seems less nice I think.

Reply:
Ok, no problem then 👍